src/components/RightSidebar/RightSidebar.vue

```diff
@@ -260,28 +260,11 @@ export default {
 	display: flex;
 }
 
-/**
- * Replace padding with margin to make the participant list scroll
- * properly behind the top field
- */
 #tab-participants {
-	margin-top: 10px;
-	padding-top: 0;
-	outline: none; /* remove the weird border that appears on focus */
-}
-
-/* Force scroll bars in tabs content instead of in whole sidebar. */
-::v-deep .app-sidebar-tabs {
-	height: calc(100% - 80px);
-
-	&__content {
-		overflow: hidden;
+	padding-top: 0px;
```

Comment:
I still don't see why that is necessary :thinking:

Reply:
after my PR:
<img width="224" alt="image" src="https://user-images.githubusercontent.com/277525/97449404-e9b55880-1931-11eb-901b-83572f8fd186.png">

if I remove padding-top: 0
<img width="224" alt="image" src="https://user-images.githubusercontent.com/277525/97449450-f76ade00-1931-11eb-9caf-b0fc48f88d3a.png">

so I need to compensate by removing the 8px from the content (the top green border)
<img width="238" alt="image" src="https://user-images.githubusercontent.com/277525/97449592-208b6e80-1932-11eb-8821-1afb582bec9d.png">

Reply:
adjusted and squashed based on more hints from @skjnldsv, thanks!